### DISCUSSION TITLE
Strange rendering of diff code block

### DISCUSSION THREAD

**Discussion by mrdrivingduck** [vuepress/core#1515] [Q&A]
When I try to render a diff code block in VuePress 2, the Markdown code and the expected appearance looks like:

```diff
-polar_hostid=1
+polar_hostid=2
-synchronous_standby_names='replica1'
```

However, when it is rendered by VuePress, the actual apperance looks like:

![image](https://github.com/vuepress/core/assets/32560442/15873048-e911-4e1b-a61d-488ddfd34bf7)

How does that come? I have tested on hljs and it seems to return right rendered format:

![image](https://github.com/vuepress/core/assets/32560442/93525313-f08e-4a46-8ed7-aa845eb067eb)

**Comment by meteorlxy** [comment]
We also use diff in our docs:

<img width="811" alt="image" src="https://github.com/vuepress/core/assets/18205362/26378e20-114a-455a-8136-8c3c45b61a04">

You'd better provide a reproduce repo.
